webapp/settings.py

```diff
@@ -17,33 +20,51 @@
 # (see: http://stackoverflow.com/a/15383766/613540)
 SECRET_KEY = 'SECRET_KEY_INSECURE_PLACEHOLDER'  # !! CHANGE ME !!
 
-# See https://docs.djangoproject.com/en/dev/ref/contrib/
-INSTALLED_APPS = ['django_versioned_static_url']
-
 ALLOWED_HOSTS = ['*']
+DEBUG = False
 
-MIDDLEWARE_CLASSES = []
-
-ROOT_URLCONF = 'webapp.urls'
-WSGI_APPLICATION = 'webapp.wsgi.application'
 LANGUAGE_CODE = 'en-us'
 TIME_ZONE = 'UTC'
 USE_I18N = False
 USE_L10N = False
 USE_TZ = False
-STATIC_ROOT = 'static'
+
+WSGI_APPLICATION = 'webapp.wsgi.application'
+ROOT_URLCONF = 'webapp.urls'
+APPEND_SLASH = False
+REMOVE_SLASH = True
+STATIC_ROOT = "static"
 STATIC_URL = '/static/'
-STATICFILES_FINDERS = ['django_static_root_finder.StaticRootFinder']
-TEMPLATE_DIRS = ['templates']
 
-# See http://tinyurl.com/django-context-processors
-TEMPLATE_CONTEXT_PROCESSORS = [
-    "django.core.context_processors.static",  # {{ STATIC_URL }}
-    "django_asset_server_url.asset_server_url",  # {{ ASSET_SERVER_URL }}
+WHITENOISE_MAX_AGE = 31557600
```

Comment:
Actually, since this is an open-source project so might well be used outside our team, I think it's too dangerous to assume people will handle updates to static files responsibly. So I'm making a call - I think we should remove this line. Unless you object?